src/main/java/org/openhab/automation/jrule/internal/engine/JRuleEngine.java

```diff
@@ -470,12 +470,11 @@ private void invokeRuleInternal(JRuleExecutionContext context, JRuleEvent event)
                 method.invoke(rule);
             }
         } catch (IllegalAccessException | IllegalArgumentException | SecurityException e) {
-            JRuleLog.error(logger, context.getMethod().getName(), "Error {}", e);
+            JRuleLog.error(logger, context.getMethod().getName(), "Error calling rule method: {}",
```

Comment:
Should we log this as WARN instead? JRule itself works fine before and after this, it is just a rule with a problem?

Reply:
I would prefer error. My current doing is to filter for errors to find errors :D
It's still an error something did not work and thats a problem. Warn will not cause any problems.

Reply:
No, and it does not cause any problems for _JRule_ itself, just unable to execute the specific rule (user code). Maybe change your log level to WARN to root out other problems? :) 

WDYT @seaside1 ?

Reply:
Yeah that's right that it's no direct error in JRule, but it's still an error. I want to find all errors. Maybe we can assure to log all user/usage errors with an extra logger?

Reply:
Something like
`JRuleLog.debug(LoggerFactory.getLogger(JRule.class), JRule.getRuleLogName(), message, parameters);`
but i am not happy with this because it obfuscates the real logging class.
As additional argument: from the JRule class every user can call logError which will log an error as well.